package.json

```diff
@@ -50,10 +50,10 @@
     "glob": "^7.1.1",
     "ip": "^1.1.5",
     "load-json-file": "^2.0.0",
-    "webdriverio": "^4.9.8",
     "wdio-mocha-framework": "^0.5.11",
+    "wdio-static-server-service": "^1.0.0",
```

Comment:
I feel like `wdio-static-server-service` should be a devDependency

Reply:
you're right, it should be

Reply:
Also side note, do we want to also list webdriverio, wdio-mocha-framework & wdio-visual-regression-service as both devDependencies and peerDependencies so consumers can set their version?